runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabQuantityOverlay.java

```diff
@@ -0,0 +1,75 @@
+package net.runelite.client.plugins.banktags.tabs;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.Point;
+import net.runelite.api.widgets.WidgetItem;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.plugins.banktags.BankTagsConfig;
+import net.runelite.client.plugins.banktags.TagManager;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.WidgetItemOverlay;
+import net.runelite.client.util.QuantityFormatter;
+
+public class TabQuantityOverlay extends WidgetItemOverlay
+{
+	private final Client client;
+
+	@Inject
+	private ItemManager itemManager;
+
+	@Inject
+	private TabInterface tabInterface;
+
+	@Inject
+	private TagManager tagManager;
+
+	@Inject
+	private BankTagsConfig bankTagsConfig;
+
+	@Inject
+	TabQuantityOverlay(Client client, TabInterface tabInterface, TagManager tagManager, BankTagsConfig bankTagsConfig)
+	{
+		this.client = client;
+		this.tabInterface = tabInterface;
+		this.tagManager = tagManager;
+		this.bankTagsConfig = bankTagsConfig;
+		showOnBank();
+	}
+
+	@Override
+	public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWidget)
+	{
+
+		if (bankTagsConfig.displayQuantity() == false || tabInterface.getActiveTagTab() == null)
+		{
```

Comment:
Should probably change the first part  `bankTagsConfig.displayQuantity() == false` to  `!bankTagsConfig.displayQuantity()`